lib/ddtrace/tracer.rb

```diff
@@ -161,6 +161,13 @@ def active_span
       @buffer.get()
     end
 
+    # stats returns a dictionary of stats about the writer.
```

Comment:
Are we going to keep or remove this one? I'm fine with that if it's useful, unless it impacts client performance or makes use of locks.